internal/engine/filter/convert_test.go

```diff
@@ -363,3 +363,19 @@ func TestActions(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expected, got)
 }
+func TestDoubleQuoteError(t *testing.T) {
```

Comment:
This is missing a newline between this function and the one above. It makes me think you're not formatting with `gofmt`.

```
$ go fmt ./...
```

Depending on your editor, you should be able to configure it to do it for you at every save.

Reply:
Yes, I'm still working on getting everything in the IDE configured and figuring out how the VS code debugger works...